jetty-core/jetty-websocket/jetty-websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/messages/AbstractMessageSink.java

```diff
@@ -18,14 +18,82 @@
 
 import org.eclipse.jetty.websocket.core.CoreSession;
 
+/**
+ * <p>Partial implementation of {@link MessageSink}.</p>
```

Comment:
Partial suggests partial message
```suggestion
 * <p>Abstract implementation of {@link MessageSink}.</p>
```